apps/src/music/views/MiniMusicPlayer.tsx

```diff
@@ -35,15 +37,22 @@ const MiniPlayerView: React.FunctionComponent<MiniPlayerViewProps> = ({
   const [currentProjectId, setCurrentProjectId] = useState<string | undefined>(
     undefined
   );
+  const {userId, userType, signInState} = useAppSelector(
+    state => state.currentUser
+  );
+  const analyticsReporter = useMemo(() => new AnalyticsReporter(), []);
 
-  // Setup library and workspace on mount
+  // Setup library and workspace, and analyticsReporter on mount
   const onMount = useCallback(async () => {
     setUpBlocklyForMusicLab();
     workspaceRef.current.initHeadless();
     const library = await loadLibrary(libraryName);
     MusicLibrary.setCurrent(library);
     setIsLoading(false);
-  }, [libraryName]);
+    analyticsReporter.startSession().then(() => {
```

Comment:
Would be good to add some additional info to the session payload so we can differentiate this from a standard music lab session. Maybe passing an optional `sessionType: 'miniPlayer' | 'project'` here that gets added to the session end payload?

Reply:
Nice idea! Will update.

Reply:
Per [Slack discussion](https://codedotorg.slack.com/archives/C04TH8400AU/p1717691182046949), not sending a session report on mini-player since we're tracking session time in GA4 already so will remove `sessionType`.